### DISCUSSION TITLE
0 photos and videos?

### DISCUSSION THREAD

**Discussion by blooumpy** [icloud-photos-downloader/icloud_photos_downloader#967] [Q&A]
I'm able to authenticate with the username and go through the 2fa successfully but when I run the following command, it says I have 0 photos and videos, however I can confirm the account I'm using has photos and videos in it.  Any ideas why?

screenshot below:
 
![image](https://github.com/user-attachments/assets/6aae813c-ddb1-4b85-ae6a-699d4eb4a9b0)

**Comment by AndreyNikiforov** [comment]
Are you using shared ~~folders~~ libraries in icloud? [Personal lib is used by default](https://icloud-photos-downloader.github.io/icloud_photos_downloader/reference.html)

**Reply by blooumpy** [reply]
I just ran --list-libraries and the result was "PrimarySync" (which I assume is the name of the library?  Then I ran it with --library PrimarySync and it came up with the same result (0 original photos and videos) like in the screenshot.   Any other ideas?  .. and did I do that right?  

![image](https://github.com/user-attachments/assets/f0bc444c-d961-432a-8fb8-58f8d972eada)

**Reply by AndreyNikiforov** [reply]
It you have only one, personal, library, then you do not have to specify it. Is that the only library visible in icloud.com web ui? Do you see photos in icloud.com web UI? What [other] parameters are you specifying in command line?

**Reply by blooumpy** [reply]
No other parameters. the screenshot above is exactly what I am using.  i am able to see all my photos and videos when I log into photos.icloud.com.  

Ok, so I guess this isn't normal, I just wanted to make sure I wasn't missing something.

**Reply by AndreyNikiforov** [reply]
Basic account should work fine (no other reports and mine is downloading as we speak), so need to understand how your set up is different from everybody else. Shared lib was the first idea, because there were reports of some [random?] anomalies. No other good ideas from me so far.
